pkg/infra/log/log.go

```diff
@@ -433,9 +442,55 @@ func ReadLoggingConfig(modes []string, logsPath string, cfg *ini.File) error {
 		configLoggers = append(configLoggers, handler)
 	}
 
+	var err error
+	root.gokitLogActivated, err = isNewLoggerActivated(cfg)
+	if err != nil {
+		return err
+	}
 	if len(configLoggers) > 0 {
 		root.initialize(configLoggers)
 	}
 
 	return nil
 }
+
+// This would be removed eventually, no need to make a fancy design.
+// For the sake of important cycle I just copied the function
+func isNewLoggerActivated(cfg *ini.File) (bool, error) {
+	section := cfg.Section("feature_toggles")
+	toggles, err := readFeatureTogglesFromInitFile(section)
+	if err != nil {
+		return false, err
+	}
+	return toggles["gokitlogActivated"], nil
+}
+
+func readFeatureTogglesFromInitFile(featureTogglesSection *ini.Section) (map[string]bool, error) {
```

Comment:
This feels a bit ugly, but understand why it's needed. Another more simple approach might be to include a configuration property to [log]. WDYT?

Reply:
🤔 I like that idea. Since we can’t use the feature toggles with all their uh features, it seems like a property in “log” could give us what we want in a simpler fashion.

Reply:
Since the feature toggle would be used by Hosted Grafana and this code would be removed in 2 weeks... I would prefer to keep it as a feature toggle if there is no strong rejection on it. Otherwise, HG has to adapt the way we "activate" this new log for testing, not sure for a temporary code is worthy.

Reply:
👋 Hi from HG 😁

A feature toggle would be ideal here - we already have the mechanisms in place to easily toggle them on/off for at the instance, cell, or global level. Having a configuration element to modify - especially for such a short period as this - would require more effort and coordination on our side.

Reply:
how about I name the feature toggle as newLoggerAsDefault, when it is present I use the new logger and if not I use the old one. Simple binary decision :).

Reply:
😂

Yes, I am not expecting you to be the ones to complain about that. I talked with Ying and we're going forward with the one feature toggle for 8.5 and dropping it for Grafana 9.

Reply:
I think here we could use newlog feature toggle for testing and activation to new log format on HG, and for v9 I would have an oldlog feature toggle to make sure that we can fallback, but for HG since it would not be needed since we would already tested for v8. It would just for OSS user instead. We can keep it as feature toggle for easy operations. :)